src/battle-log.ts

```diff
@@ -126,7 +126,13 @@ class BattleLog {
 			return;
 
 		case 'unlink':
-			this.hideChatFrom(toId(args[2] || args[1]));
+			const user = toId(args[2]) || toId(args[1]);
+			const $messages = $('.chatmessage-' + user);
```

Comment:
jQuery is not a supported dependency in battle logs.

Reply:
Look at the header comment:

https://github.com/Zarel/Pokemon-Showdown-Client/blob/bf09524187f2f7d997ed0ef6f3d5dd7e604c7349/src/battle-log.ts#L4-L10

Reply:
I'd suggest manually iterating child nodes.

Reply:
Like, look at the source code for `hideChatFrom` and adapt that.

Reply:
Also, you can't use `getElementsByClassName` because that's not supported by all supported browsers.